docs/pages/guides/recipes/data-modeling/period-over-period.mdx

```diff
@@ -17,6 +17,12 @@ _previous period_.
 these `rolling_window` measures and uses them in a calculation, e.g.,
 divides or subtracts them.
 
+<WarningBox>
+
+Please note that Redshift does not support intervals of type month or year.
+
+</WarningBox>
+
```

Comment:
@morgan-at-cube I don't think we need to put this warning on this particular page. If we do, then we'd probably need to have it at every page mentioning `rolling_window` which is probably not the way to go.